### PR TITLE
Final prep for publication

### PR DESCRIPTION
**Pull request by alan-edi**
This makes the following changes, as final prep for publishing to the Gradle Plugin Portal:
- In `build.gradle`:
  - Change `website` & `vcsUrl` to this public `Esri` repo.
  - Delete the `publishing` block that was used to publish to a local repo to check how the artifacts look.
- Update Gradle version from 8.4 to 8.5.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+networkTimeout=10000
```
Comment (mbcoder) [inline gradle/wrapper/gradle-wrapper.properties:4]: Is this needed?  Looks like the default is 10000?  https://docs.gradle.org/7.6/release-notes.html

**Review by mbcoder — APPROVED** [review]
This looks okay.  The only thing I wasn't sure about was the networkTimeout.  I don't think it matters either way, so I've approved.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (alan-edi) [reply gradle/wrapper/gradle-wrapper.properties:4]: @mbcoder Good question. `networkTimeout` and `validateDistributionUrl` were added when I updated the gradle version by running `./gradlew wrapper --gradle-version 8.5`. Looking at the various `gradle-wrapper.properties` files in our java development repo, they all have:
```
networkTimeout=10000
validateDistributionUrl=true
```
It looks like `networkTimeout=10000` came in with Gradle version 7.6 and `validateDistributionUrl=true` came in with 8.2.

In light of that, would you agree it makes sense to leave them in?

**Comment by alan-edi** [comment]
@mbcoder Thanks for your feedback on the new gradle wrapper properties. Merging.

**Pull request merged**